smartsim/log.py

```diff
@@ -68,6 +89,78 @@ def _get_log_level() -> str:
     return "info"
 
 
+def get_exp_log_paths() -> t.Tuple[t.Optional[pathlib.Path], t.Optional[pathlib.Path]]:
```

Comment:
A couple of docstrings in this file miss the `:param:`/`:type:`/`:returns:`/`:rtype:` lists. Can you please add them to make rendered docs more consistent?

Reply:
Actually, thanks for reminding me of the impending check-style-pocalypse! Will do